src/com/projectkorra/projectkorra/util/ParticleEffect.java

```diff
@@ -40,6 +40,7 @@
 	FALLING_DUST (Particle.FALLING_DUST),
 	FIREWORKS_SPARK (Particle.FIREWORKS_SPARK),
 	FLAME (Particle.FLAME),
+	SOUL_FIRE_FLAME (Particle.SOUL_FIRE_FLAME),

```

Comment:
[For future reference](https://i.imgur.com/W5TITGe.png), wanna make sure we add the other particles too

Reply:
Sorry, I was only focusing on blue fire for this.